eo-parser/src/main/resources/org/eolang/parser/parse/stars-to-tuples.xsl

```diff
@@ -53,7 +53,6 @@
           <xsl:element name="o">
             <xsl:attribute name="base" select="'Q.org.eolang.number'"/>
             <xsl:element name="o">
-              <xsl:attribute name="base" select="'Q.org.eolang.bytes'"/>
```

Comment:
@volodya-lombrozo I think there's so need to remove this line, instead you can add one more inner `<o>` element here with `@hex` attribute. It'll be still caught by `StHex` but won't be caught by `explicit-data.xsl`

Reply:
@maxonfjvipon Actually, it was my first attempt. I tried exactly, what you suggested. However, it wasn't possible due to the order of transformations. I've tried to explain it in [the first puzzle](https://github.com/objectionary/eo/pull/4041/files#diff-8b2d766e253403f4d30fa362667f952a955eccc0167a090f2ff544279610f0e4R82).
 
Now, I tried it again with all the new changes (since we have rearranged transformations). And all the tests in `eo-parser` pass now, indeed. However, 7 tests are failed in `eo-maven-plugin`. You can try running them yourself. 

So, I guess, for now, it's the only viable solution.